runtime/common/src/lib.rs

```diff
@@ -805,6 +805,34 @@ macro_rules! impl_config_traits {
                     ProxyType::Staking => matches!(c, RuntimeCall::ParachainStaking(..)),
                     #[cfg(not(feature = "parachain"))]
                     ProxyType::Staking => false,
+                    ProxyType::CreateMarket =>
+                        matches!(c,
+                            RuntimeCall::PredictionMarkets(zrml_prediction_markets::Call::create_market {..}) |
+                            RuntimeCall::PredictionMarkets(zrml_prediction_markets::Call::create_cpmm_market_and_deploy_assets {..})
+                        ),
+                    ProxyType::ReportOutcome =>
+                        matches!(c,
+                            RuntimeCall::PredictionMarkets(zrml_prediction_markets::Call::report {..})
+                        ),
+                    ProxyType::ProvideLiquidity =>
+                        matches!(c,
+                            RuntimeCall::Swaps(zrml_swaps::Call::pool_join {..}) |
+                            RuntimeCall::Swaps(zrml_swaps::Call::pool_join_with_exact_asset_amount {..}) |
+                            RuntimeCall::Swaps(zrml_swaps::Call::pool_join_with_exact_pool_amount {..}) |
```

Comment:
These _single-asset_ operations are kind of funky in that they execute trades, so they should probably not be here. It's fine to remove them completely, we don't want single-asset ops for prediction markets anyways.

Reply:
```suggestion
```

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1038/commits/b80ae75c2da1bc7ce687cb5e2a5aa6d1aef2d37c